es-ds-components/components/es-form-msg.vue

```diff
@@ -0,0 +1,69 @@
+<script setup lang="ts">
+const props = defineProps({
+    show: {
+        type: Boolean,
+        default: false,
+    },
+    timeout: {
+        type: Number,
+        default: 20,
+        required: false,
+    },
+    variant: {
+        type: String,
+        default: 'danger',
+        required: false,
+        validator: (val: string) => ['danger', 'success', 'primary'].includes(val),
+    },
+});
+
+const emit = defineEmits(['hidden']);
+
+const hide = () => {
+    emit('hidden');
+}
+
+watch(() => props.show, (newValue, oldValue) => {
+    if (newValue === true && oldValue === false) {
```

Comment:
Could simplify this to `if (newValue && !oldValue) {`

Reply:
They actually don't quite do the same thing, but I agree!